### DISCUSSION TITLE
Firmware of 2023.05.13?

### DISCUSSION THREAD

**Discussion by matt1aw** [ludwig-v/wireless-carplay-dongle-reverse-engineering#174] [General]
Does anyone have it?

Ref.: https://www.facebook.com/groups/carlinkitservices/posts/1632266190609685/?__cft__[0]=AZUqnIfjHN2rmeO9jKFdwMg7DOscSNow2WWGiq_p4MZJc4N4-QZY1j70qtg0UlohASvYCaxfF87Myzkp0IVGUGdaRcYKP1PJh6PC_xEMNRDEw3iz2MBQO-L2-50-LS5EQPz41uM0Stt55pWDYxROXKb2&__tn__=%2CO%2CP-R

**Comment by ludwig-v** [comment]
[Here ?](https://cdn.shopify.com/s/files/1/0618/3783/6515/files/U2W_AUTOKIT_Update_1.img?v=1685437691)

To rename in U2W_AUTOKIT_Update.img
Can you report the full version if you install it (from 192.168.50.2) ? (2023.05.13.XXXX)

**Reply by matt1aw** [reply]
I think I have already done a successful flash on your link provided. But the the software ver. seems remain unchanged, i.e. 2022.12.14.1349.

The MD5 file hash of the link is 72b896507e7557509de7dd240ae681da, which is same as the MD5 of 2022.12.14.1349 as provided by [Carlinkit](https://www.facebook.com/groups/carlinkitservices/posts/1634001440436160/?comment_id=1634028090433495&__cft__[0]=AZWWARKa7gZ24sA5h_5zT-5_xUzXCj6VdZ2eOkjjKHWxO6gTjy5tlV4HCs4qSMlRvz0kG4__i7K3b4PIxYENn2sssHobp5mbEJMNOIdiuddqmUdSnG0VgrchPxX_SptlhMYcpoeJrlK7pS7Mr7jbIW8K&__tn__=R]-R). Orz

**Reply by matt1aw** [reply]
[This](https://carlinkit.co.kr/2023/06/14/3-0-2023-06-14-업데이트-로그/) Carlinkit Korean website provided a firmware of 2023.05.13.

I have tried installing it but failed. The green and red LEDs just flashed a few times and stopped. No change in the software ver..
